.github/workflows/test.yml

```
@@ -215,3 +215,17 @@ jobs:
       saves_to_disk: true
       disk_suffix: tip
       height_grep_text: 'finished initial sync to chain tip, using gossiped blocks sync_percent=100.* current_height=Height'
+
+  lightwalletd-rpc-test:
+    if: ${{ github.event.inputs.regenerate-disks != 'true' }}
+    needs: build
+    uses: ./.github/workflows/gcp-test-deploy.yml
+    with:
+      app_name: lightwalletd
+      test_id: fully-synced-rpc
+      test_description: Test lightwalletd RPC with a Zebra tip state
+      test_variables: '-e TEST_LWD_RPC_CALL=1 -e ZEBRA_TEST_LIGHTWALLETD=1 -e ZEBRA_FORCE_USE_COLOR=1 -e ZEBRA_CACHED_STATE_DIR_VAR=/var/cache/zebrad-cache'
```

Blocker: the environmental variable name is incorrect

```suggestion
      test_variables: '-e TEST_LWD_RPC_CALL=1 -e ZEBRA_TEST_LIGHTWALLETD=1 -e ZEBRA_FORCE_USE_COLOR=1 -e ZEBRA_CACHED_STATE_DIR=/var/cache/zebrad-cache'
```

---

I'll go fix the Rust code to be less confusing. The Rust fix doesn't block this PR.

---

Shouldn't the test fail if this happens? https://github.com/ZcashFoundation/zebra/runs/6298730934?check_suite_focus=true#step:8:219